.github/workflows/release.yml

```diff
@@ -18,6 +18,8 @@ jobs:
     - name: Unzip wheels asset
       run: |
         unzip wheels.zip -d dist
+    - name: Download tar.gz.b64
```

Comment:
Could use a more descriptive name.

Reply:
The CI is broken for an unknown reason, see https://github.com/Farama-Foundation/Arcade-Learning-Environment/issues/515 for more info